index.js

```diff
@@ -14,7 +14,12 @@ const yaml = require('js-yaml');
 const check = require('check-type').init();
 
 const YamlValidatore = function YamlValidatore(options) {
-  this.options = options;
+  this.options = Object.assign({
```

Comment:
Yeah! Creating tests do help to identify trouble spots in the code :)

I'm intending to put in a few more tests, then maybe you can merge this PR?